### PR TITLE
Fix duplicate results appearing when there were multiple annotations

### PR DESCRIPTION
**Pull request by david-smith-ft**
Regardless of predicates used, it should only show one result per content.

In addition, switched to using dep instead of govendor as a test.  Also, simplified running tests in CircleCI based on changes in recent go versions (ignoring vendor folder for go test).

### CODE REVIEW

**Comment by david-smith-ft** [comment]
[Public Content-by-Concept API returns duplicate IDs](https://trello.com/c/jT3V1Tcf/444-public-content-by-concept-api-returns-duplicate-ids)

File: cypher.go [hunk 1]
```diff
@@ -3,8 +3,8 @@ package main
 import (
 	"github.com/Financial-Times/neo-model-utils-go/mapper"
 	"github.com/Financial-Times/neo-utils-go/neoutils"
-	log "github.com/Sirupsen/logrus"
 	"github.com/jmcvetta/neoism"
+	log "github.com/sirupsen/logrus"
```
Comment (izzyblues) [inline cypher.go]: Did you want to use `github.com/Financial-Times/go-logger` or `github.com/sirupsen/logrus`?

File: .circleci/config.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.7
+      - image: circleci/golang:1.9
       - image: neo4j:3.1.0-enterprise
```
Comment (izzyblues) [inline .circleci/config.yml]: I think this isn't the current version of neo4j that we run in Prod, right? 😅

File: cypher.go (hunk 1 above)
Comment (david-smith-ft) [reply cypher.go]: I had to change Sirupsen to sirupsen, so I was going to upgrade go-logger at the same time but decided not to.  The import is probably where I missed reverting something.

File: .circleci/config.yml (hunk 2 above)
Comment (david-smith-ft) [reply .circleci/config.yml]: Correct, we use 3.2.7

File: Gopkg.lock [hunk 3]
```diff
@@ -0,0 +1,241 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```
Comment (khatton-ft) [inline Gopkg.lock]: Sounds like this file probably shouldn't be checked in, as the Dockerfile calls `dep ensure` so its contents can't be relied upon.

File: app.go [hunk 4]
```diff
@@ -9,12 +9,13 @@ import (
...
 	"github.com/Financial-Times/base-ft-rw-app-go/baseftrwapp"
 	fthealth "github.com/Financial-Times/go-fthealth/v1_1"
+	"github.com/Financial-Times/go-logger"
 	"github.com/Financial-Times/http-handlers-go/httphandlers"
 	"github.com/Financial-Times/neo-utils-go/neoutils"
 	status "github.com/Financial-Times/service-status-go/httphandlers"
-	log "github.com/Sirupsen/logrus"
 	"github.com/gorilla/mux"
 	"github.com/jawher/mow.cli"
+	_ "github.com/joho/godotenv/autoload"
```
Comment (khatton-ft) [inline app.go]: Having looked at the project readme I don't believe this import is actually doing anything.

File: Gopkg.lock (hunk 3 above)
Comment (izzyblues) [reply Gopkg.lock]: Done!

File: app.go (hunk 4 above)
Comment (david-smith-ft) [reply app.go]: This import allows a `.env` file to be used to define environment variables, which is especially useful when developing.

**Pull request merged**